transport.go

```diff
@@ -66,12 +129,32 @@ func (c *connManager) createConn(network, host string) (net.PacketConn, error) {
 	return net.ListenUDP(network, addr)
 }
 
+func (c *connManagers) AddListener(network string, addr *net.UDPAddr, conn net.PacketConn) error {
+	if connManager, ok := c.managers[network]; ok {
+		return connManager.AddListener(network, addr, conn)
```

Comment:
Nit: Feeding the network through here just to be thrown away seems funny. We don't _have_ to use the exact same interface for the connManager and the connManagers.

(although I don't mind much either way)